examples/02-gravity/plot_inv_grav_tiled.py

```diff
@@ -26,7 +26,7 @@
 try:
     from SimPEG import utils
     from SimPEG.utils import plot2Ddata
-except:
+except ImportError:
```

Comment:
Same comment as the gravity_driver.py file.

I suspect that these were do to someone trying to simulatenously support multiple versions of SimPEG